examples/data-sources/cockroach_person_user/cockroach_person_user.tf

```diff
@@ -0,0 +1,7 @@
+variable "email_address" {
+  type = string
+}
+
+data "cockroach_person_user" "cockroach" {
```

Comment:
This `cockroach_person_user` name might be a little confusing since it's unclear whether you're talking about a user in the database or the cloud. Not sure what the right naming might look like, though.

Reply:
Yeah, I'm not sure what better naming would look like either. If we hear customers are getting confused, we can always add more detail to to data source's description.